sympy/integrals/integrals.py

```diff
@@ -394,6 +395,8 @@ def doit(self, **hints):
             return S.Zero
 
         # now compute and check the function
+        if isinstance(self.function, Sum):
+            return Sum(integrate(self.function.function, self.variables[0]), self.function.limits)
```

Comment:
On #7827 @asmeurer mentioned that one should "go inside summations that don't have the integration variable in the limits". You haven't checked for that here. e.g., perhaps he was thinking of something like this: `integrate(Sum(x, (x, 1, y)), y)`:

```python
>>> self = Integral(Sum(x,(x,1,y)),y)
>>> Sum(integrate(self.function.function, self.variables[0]),
... self.function.limits)
Sum(x*y, (x, 1, y))
>>> _.doit().expand()
y**3/2 + y**2/2
```

but compare to 

```python
>>> self.doit().expand()
y**3/6 + y**2/4
```